lib/saml/SamlIdpUrlBuilder.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var njwt = require('njwt');
+var uuid = require('node-uuid');
+
+function SamlIdpUrlBuilder(application) {
+  this.application = application;
+  this.secret = application.dataStore.requestExecutor.options.client.apiKey.secret;
+}
+
+SamlIdpUrlBuilder.prototype._getServiceProvider = function _getServiceProvider(callback) {
+  this.application.getSamlPolicy(function (err, samlPolicy) {
+    if (err) {
+      return callback(err);
+    }
+
+    samlPolicy.getServiceProvider(callback);
+  });
+};
+
+SamlIdpUrlBuilder.prototype._buildInitializationUrl = function _buildInitializationUrl(initEndpointUrl, accessToken) {
+  return initEndpointUrl + '?accessToken=' + encodeURIComponent(accessToken);
```

Comment:
By bad, something went wrong when I used the Node proxy. When I used the API directly everything worked.

The optional params to the URL still needs to be fixed though :)